purchase_request_product_usage/wizards/purchase_request_line_make_purchase_order.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+
+from odoo import api, fields, models
+
+
+class PurchaseRequestLineMakePurchaseOrder(models.TransientModel):
+    _inherit = "purchase.request.line.make.purchase.order"
+
+    @api.model
+    def _prepare_purchase_order_line(self, po, item):
+        res = super(PurchaseRequestLineMakePurchaseOrder, self).\
+            _prepare_purchase_order_line(po, item)
+        res.update(usage_id=item.usage_id.id)
+        return res
+
+    @api.model
+    def _prepare_item(self, line):
+        res = super(PurchaseRequestLineMakePurchaseOrder, self)._prepare_item(
+            line)
+        res.update(usage_id=line.usage_id.id)
```

Comment:
@aheficent `return res.update(usage_id=line.usage_id.id)`

Reply:
This causes error because update returns null